### PR TITLE
We can now delegate again like before

### PR DESCRIPTION
**Pull request by annando**
There is now a possibility to do the delegation like it was done before. When a user record has got a ```parent-uid``` that isn't equal zero, then that parent user can completely manage that user account.

Currently a web frontend for this setting is missing. Hopefully someone does this in a nice way. :-)

### CODE REVIEW

File: include/security.php [hunk 1]
```diff
@@ -146,7 +169,7 @@ function authenticate_success($user_record, $login_initial = false, $interactive
 
 		// Set the login date for all identities of the user
 		dba::update('user', ['login_date' => DateTimeFormat::utcNow()],
-			['password' => $master_record['password'], 'email' => $master_record['email'], 'account_removed' => false]);
+			['parent-uid' => $master_record['uid'], 'account_removed' => false]);
```
Comment (MrPetovan) [inline include/security.php]: This only works if one logs in as the parent account. If logged with a managed accound, no rows will be updated.

Comment (annando) [reply include/security.php]: Yes. But I don't see it as a problem.

Comment (MrPetovan) [reply include/security.php]: Ok then.

**Comment by tobiasd** [comment]
> Currently a web frontend for this setting is missing. Hopefully someone does this in a nice way. :-)

Is there an issue for this blocking the 3.6 milestone?

**Comment by annando** [comment]
I think so.

**Comment by tobiasd** [comment]
Do you mean #4319 ? then please add a comment to update the description of the situation ;-)

**Comment by annando** [comment]
Oh, I misunderstood. I don't know of a matching issue.

**Comment by MrPetovan** [comment]
> then please add a comment to update the description of the situation ;-)

Done.

**Pull request merged**